### PR TITLE
adds http examples using curl, curlite, and guzzle

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

**Comment by bshaffer** [comment]
Related: [this page](https://cloud.google.com/appengine/docs/php/outbound-requests) needs to be updated to include code samples and a link to [this article](https://cloud.google.com/appengine/docs/php/runtime#PHP_cURL_support), as the existing page is not very useful.

File: appengine/standard/http/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# HTTP Requests & Google App Engine
+
+This sample application demonstrates how to make [HTTP Requests with Google App Engine](https://cloud.google.com/appengine/docs/php/outbound-requests).
+
+## Setup
+
+Before running this sample:
```
Comment (tmatsuo) [inline appengine/standard/http/README.md]: `Setup` is almost empty. Can you just remove it?

File: appengine/standard/http/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+- Install [`composer`](https://getcomposer.org)
+- Install dependencies by running:
+
+```sh
+composer install
+```
+
+## Deploy to App Engine
+
+**Prerequisites**
```
Comment (tmatsuo) [inline appengine/standard/http/README.md]: Move this prerequisites to the upper one.

**Comment by tmatsuo** [comment]
LGTM except for some nits in README

**Comment by bshaffer** [comment]
@tmatsuo done! Thanks 💃

**Pull request merged**